CourseApp/RPG/Abstract/Player.cs

```diff
@@ -0,0 +1,59 @@
+namespace CourseApp
+{
+    using System;
+
+    public abstract class Player : IPlayer
+    {
+        private readonly Random random = new Random();
```

Comment:
:no_bell: In .NET 6 use may use `Random.Shared` without creating new instance (e.g. `Random.Shared.Next(0, 10)`)